packages/forklift-console-plugin/src/modules/ProvidersNG/utils/validators/provider/vsphereProviderValidator.ts

```diff
@@ -0,0 +1,23 @@
+import { V1beta1Provider } from '@kubev2v/types';
+
+import { validateContainerImage, validateK8sName, validateURL } from '../common';
+
+export function vsphereProviderValidator(provider: V1beta1Provider) {
+  const name = provider?.metadata?.name;
+  const url = provider?.spec?.url || '';
+  const vddkInitImage = provider?.spec?.settings?.['vddkInitImage'];
+
+  if (!validateK8sName(name)) {
+    return new Error('invalided kubernetes resource name');
+  }
+
+  if (!validateURL(url)) {
+    return new Error('invalided URL');
+  }
+
+  if (vddkInitImage !== '' || !validateContainerImage(vddkInitImage)) {
```

Comment:
```suggestion
  if (vddkInitImage !== '' && !validateContainerImage(vddkInitImage)) {
```

the logic is if not emapty and not valid then show error